src/core_plugins/kibana/public/visualize/editor/agg_params.js

```diff
@@ -117,6 +104,19 @@ uiModules
         $compile($aggParamEditors)($aggParamEditorsScope);
       });
 
+      // this will contain the controls for the schema (rows or columns?), which are unrelated to
```

Comment:
Would you mind wrapping this and the rest of the initialization in functions, then calling them in the right order and describing in comments why the order is important?